package.json

```
@@ -64,7 +64,7 @@
   },
   "packageManager": "yarn@4.2.2",
   "scripts": {
-    "build:all": "yarn workspaces foreach --include './ui/{npm-content,uppy,legacy,app}' -Ap run build-if-changed && yarn workspaces foreach --include './ui/{scss,guest}' -Ap run build-if-changed",
+    "build:all": "yarn workspaces foreach --include './ui/uppy' -A run build-if-changed && yarn workspaces foreach --include './ui/{npm-content,legacy,app}' -Ap run build-if-changed && yarn workspaces foreach --include './ui/{scss,guest}' -Ap run build-if-changed",
```

The foreach doesn't make sense for a single workspace. I think there's a way of running for a single. Try something like...
```suggestion
    "build:all": "yarn workspace @craftercms/uppy build-if-changed && ...",
```
```suggestion
    "build:all": "yarn workspace ./ui/uppy build-if-changed && ...",
```

...or adding `run` to those before the script name

---

Just confirmed `yarn workspace @craftercms/uppy build-if-changed` works.